### PR TITLE
updated the application icon

### PR DESCRIPTION
**Pull request by AakashRaj20**
**Notes for Reviewers**

This PR fixes #
updated the new svg for application Icon also added the color prop and also added all the neccessary prop

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/package-lock.json
Comment (leecalcote) [inline ui/package-lock.json:1]: @AakashRaj20 please remove changes to this file.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by Chadha93** [comment]
@AakashRaj20 This comment is not directly related to your changes. 
@Abhishek-kumar09 @harkiratsm Replacing the app's icon here in Meshery UI is overriding the app's icon in Meshery Extensions (making disappear). I checked in extensions, we are not referencing this SVG from Meshery anywhere. I observed that when the Meshery drawer is active, the apps icons appears to be fine in extensions. Capability Registry might be affecting here on show - https://github.com/meshery/meshery/blob/master/ui/components/Navigator.js#L429

**Comment by leecalcote** [comment]
@UsmanDev09  has this already been taken care of by @duckling69 ?

**Comment by UsmanDev09** [comment]
I am not aware of @duckling69 involvement in this issue but I will update you. @leecalcote

**Comment by UsmanDev09** [comment]
He worked on issue #7506 @leecalcote

**Pull request closed without merging**